task8/TutterApp/src/main/webapp/index.html

```diff
@@ -10,65 +10,96 @@
         <link href="http://fonts.googleapis.com/css?family=Reenie+Beanie" rel="stylesheet">
         <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
         <script src="https://kit.fontawesome.com/9b7f939454.js" crossorigin="anonymous"></script>
-        <script src="resources/js/script.js"></script>
+        <script src="resources/js/model.js"></script>
+        <script src="resources/js/view.js"></script>
+        <script src="resources/js/controller.js"></script>
     </head>
 
     <body>
     <header>
-        <div class="logo">
+        <div id="logo" class="logo">
             <img src="resources/images/logo.svg" class="logo">
             <p class="logo">Tutter</p>
         </div>
 
-        <div class="log-in-out" id="log-in-out">nikmalevich <button class="log-in-out">Log out</button></div>
+        <div class="log-in-out" id="log-in-out"><span id="username"></span><button class="log-in-out" id="logInOut">Log in</button></div>
     </header>
 
     <main>
-        <div class="line"></div>
+        <div id="mainPage">
+            <div class="line"></div>
 
-        <div class="filters">
-            <p class="filters">Filters</p>
-            <label for="filterName" class="filters">name:</label>
-            <textarea id="filterName" class="filters" rows="1"></textarea>
-            <label for="filterTag" class="filters">tag:</label>
-            <textarea id="filterTag" class="filters" rows="1"></textarea>
-            <p class="date">date:</p>
-            <label for="filterDateFrom" class="date-filters">from</label>
-            <input type="date" id="filterDateFrom" name="from" value="2020-03-02" min="2020-01-01" max="2020-12-31" class="date">
-            <label for="filterDateTo" class="date-filters">to</label>
-            <input type="date" id="filterDateTo" name="to" value="2020-03-02" min="2020-01-01" max="2020-12-31" class="date">
-            <button class="add-post">add Post</button>
-        </div>
+            <form class="filters" id="filterForm">
+                <p class="filters">Filters</p>
+                <label for="filterName" class="filters">name:</label>
+                <input id="filterName" type="text" class="filters">
+                <label for="filterTag" class="filters">tag:</label>
+                <input id="filterTag" type="text" class="filters">
+                <p class="date">date:</p>
+                <label for="filterDateFrom" class="date-filters">from</label>
+                <input type="date" id="filterDateFrom" name="from" value="2019-01-01" class="date">
+                <label for="filterDateTo" class="date-filters">to</label>
+                <input type="date" id="filterDateTo" name="to" value="2021-01-01" class="date">
+                <button class="add-post" id="addPost">add Post</button>
```

Comment:
the same class for all tags is not needed. Add class `filters` just to `form` tag.
In CSS use selectors: `form.filters` `form.filters p` `form.filters label` `form.filters input`